go/fn/go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/GoogleContainerTools/kpt-functions-sdk/go/fn
 go 1.17
 
 require (
+	github.com/GoogleContainerTools/kpt-functions-catalog/functions/go/apply-replacements v0.0.0-20220316042125-d3eb289008bc
```

Comment:
I guess we can't import this package even though it's used in the example.
Go modules don't distinguish a package's real dependencies from the dependencies in the examples. That means when we try to import the sdk package in `kpt-functions-catalog/functions/go/apply-replacements`. Go will complain about circular dependency issue. 
github.com/GoogleContainerTools/kpt-functions-catalog/functions/go/apply-replacements -> github.com/GoogleContainerTools/kpt-functions-sdk/go/fn -> github.com/GoogleContainerTools/kpt-functions-catalog/functions/go/apply-replacements

Reply:
Thank you for the catch! I have replaced the apply-replacements example with another simple example that doesn't depend on kpt-functions-catalog repo.